internal/pkg/scaffold/ansible/k8s_status.go

```diff
@@ -343,7 +343,19 @@ class KubernetesAnsibleStatusModule(KubernetesAnsibleModule):
             'changed': True
         }
 
+    #  clean_last_transition_time removes lastTransitionTime attribute from each status.conditions[*].
+    #  That is because object_contains function which is called after clean_last_transition_time, compares
+    #  if new conditions are subset of old conditions, but due to lastTransitionTime
+    #  they are never subset. New conditions should not contain lastTransitionTime too. It will be set by kubernetes.
```

Comment:
I would like to see a better explanation here. It is legal to provide lastTransitionTime when it is actually known and stable (operator logic). Most of the time we would like to rely on k8s to detect changes and set the time though.